Warui-K/politiko-api
# #163728408 Sets up environment configurations

**What does this PR do?**
Have the set up for Flask environment

**Description of Task to be completed?**
Have the following installation set up
astroid==2.1.0
atomicwrites==1.3.0
attrs==18.2.0
isort==4.3.4
lazy-object-proxy==1.3.1
mccabe==0.6.1
more-itertools==5.0.0
pluggy==0.8.1
py==1.7.0
pylint==2.2.2
pytest==4.2.0
six==1.12.0
typed-ast==1.3.1
wrapt==1.11.1

**How should this be manually tested?**
After cloning the repo cd into it and ensure you have the setup requirements
Then have the virtual environment activated source venv/bin/activate
Go to browser http://127.0.0.1:5000

**What are the relevant pivotal tracker stories?**
#163728408
